src/screens/AlertScreen.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from "react";
+import * as WebBrowser from "expo-web-browser";
+import {
+  Platform,
```

Comment:
Make sure to remove unused imports. Most IDE and text editors can be configured to highlight this.